### PR TITLE
Editing CLO section of Infra MachineSet topc

### PR DESCRIPTION
**Pull request by mburke5678**
Updating the section on creating an Infra machineset to add how to move cluster logging to an Infra node.

### CODE REVIEW

File: modules/infrastructure-moving-logging.adoc [hunk 1]
```diff
@@ -3,10 +3,84 @@
...
+Fluentd, to different nodes. You cannot move the Cluster Logging Operator pod from its installed location. 
+
+[NOTE]
 ====
-Cluster logging in {product-title} is not installed by default.
-You currently cannot move the logging component to another MachineSet.
+You should set your MachineSet to use at least 6 replicas.
 ====
+
+.Prerequisite
```
Comment (kalexand-rh) [inline modules/infrastructure-moving-logging.adoc]: Prerequisites

File: modules/infrastructure-moving-logging.adoc [hunk 2]
```diff
@@ -3,10 +3,84 @@
...
+
+* Cluster logging and Elasticsearch must be installed. These features are not installed by default.
```
Comment (kalexand-rh) [inline modules/infrastructure-moving-logging.adoc]: Logging

File: modules/infrastructure-moving-logging.adoc [hunk 3]
```diff
@@ -3,10 +3,84 @@
...
+      proxy:
+        resources: null
+      replicas: 1
+      resources: null
+    type: kibana
+
+....
+
+----
+<1> Add a `nodeSelector` parameter with the appropriate `<key>: <value>` pair to the component you want to move.
```
Comment (kalexand-rh) [inline modules/infrastructure-moving-logging.adoc]: Will you add an example that references the `node-role.kubernetes.io/infra: ""` `nodeSelector`?

**Review by kalexand-rh — COMMENTED** [review]
Two picks and a request. Will you tag QE?

**Comment by mburke5678** [comment]
@kalexand-rh I will have QE take a look. Wanted to make sure the material fit into the topic properly first. Thanks for looking!

**Comment by kalexand-rh** [comment]
@mburke5678, I'm happy to do it! If you add the example that I asked for, I think it'll fit in just fine.

File: modules/infrastructure-moving-logging.adoc (hunk 2 above)
Comment (mburke5678) [reply modules/infrastructure-moving-logging.adoc]: We have been using `cluster logging` in lowercase when talking about the feature in general.

File: modules/infrastructure-moving-logging.adoc (hunk 3 above)
Comment (mburke5678) [reply modules/infrastructure-moving-logging.adoc]: Is the `node-role.kubernetes.io/infra: ""` node selector required to your knowledge? I couldn't get it to work with that format. I tried the simple `tag=value` format, which was how QE tested the feature, and it worked.

Comment (kalexand-rh) [reply modules/infrastructure-moving-logging.adoc]: If it's not, then we should check the other node selectors.

**Comment by mburke5678** [comment]
@anpingli Can you please take a look? 
Also, @kalexand-rh [suggested I should make the node selectors in the custom resource `node-role.kubernetes.io/infra: ""`](https://github.com/openshift/openshift-docs/pull/15600#discussion_r298237466).
When I changed them from `es: yes` to `node-role.kubernetes.io/infra: ""` I got 6 new fluentd pods on the right node (the original 6 Fluentd pods remain, even after I remove the `es: yes` selector from the nodes). 
The Elasticsearch pods did not move {even after I remove the `es: yes` selector from the nodes)/
The Kibana pod moved the new nodes.

**Comment by anpingli** [comment]
@mburke5678  I think the goal is to place the ES pods into specified nodes, as ES pods require huge CPU/Memory/Disk resources.    so it is suggested adding nodeSelector for ES.   it is optional to add nodeSelector for kibana/cronjob.   It is not suggested to add nodeSeletor for damonset, as we expected the fluentd/rsyslog locate in each node

**Comment by mburke5678** [comment]
@anpingli I removed the node selector from the `spec.collector.logs.fuentd` section. And added a short note at the top "you can move the Elasticsearch pods to a separate node because of 
high CPU, memory, and disk requirements."

**Comment by anpingli** [comment]
LGTM

**Comment by mburke5678** [comment]
@ewolinetz Please take a look at this procedure to move the cluster logging components to new nodes.

File: modules/infrastructure-moving-logging.adoc [hunk 4]
```diff
@@ -3,10 +3,85 @@
...
-Cluster logging in {product-title} is not installed by default.
-You currently cannot move the logging component to another MachineSet.
+You should set your MachineSet to use at least 6 replicas.
 ====
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed. These features are not installed by default.
+
+* If needed, get the name of the Cluster Logging Custom Resource in the openshift-logging project:
```
Comment (ewolinetz) [inline modules/infrastructure-moving-logging.adoc]: the clusterlogging CR will always need to be named `instance` fyi. if it isn't the operator will print out a message and not take action on it.

**Comment by ewolinetz** [comment]
small point about the name of the CLO watched resource. Non-blocker though, lgtm.

**Pull request merged**